### PR TITLE
Update supported distros to v1.26

### PR DESCRIPTION
**Pull request by neogopher**
**What issue type does this pull request address?** (keep at least one, remove the others) 
/kind enhancement

**Please provide a short message that should be published in the vcluster release notes**
Updated supported distros to v1.26

**What else do we need to know?**    
- updated k0s,k3s,k8s chart image to latest stable release for v1.26
- updated eks chart image to latest stable release for v1.24

Closes ENG-894

### CODE REVIEW

File: charts/k3s/values.yaml [hunk 1]
```diff
@@ -150,7 +150,7 @@ syncer:
 # Virtual Cluster (k3s) configuration
 vcluster:
   # Image to use for the virtual cluster
-  image: rancher/k3s:v1.25.3-k3s1
+  image: rancher/k3s:v1.26.0-k3s1
```
Comment (matskiv) [inline charts/k3s/values.yaml:153]: There is a slightly newer stable tag - [v1.26.0-k3s2](https://hub.docker.com/layers/rancher/k3s/v1.26.0-k3s2/images/sha256-08f13056ad6755948f0ad6658ec7e04356b52351c04559956b3b9527910ffd8b?context=explore), this But from the release notes the fixes seem irrelevant to vcluster, so just FYI :)

**Review by matskiv — APPROVED** [review]
LGTM :)

File: charts/k3s/values.yaml (hunk 1 above)
Comment (neogopher) [reply charts/k3s/values.yaml:153]: Oh Yes!!   What does the 's2' in the end of the image stand-for. I saw that all our previous version images were s1, hence I took the latest 's1' image.

Comment (matskiv) [reply charts/k3s/values.yaml:153]: That is just a suffix that k3s maintainers can increment when releasing a new build of the same Kubernetes release.

Comment (neogopher) [reply charts/k3s/values.yaml:153]: Ah!! Got it. Will keep this in mind. Thanks @matskiv. :slightly_smiling_face:  :raised_hands:
